src/core/main.lua

```diff
@@ -33,23 +33,21 @@ function main ()
    zone("startup")
    require "lib.lua.strict"
    initialize()
+   local program
```

Comment:
I tried to revert that merge but it's given me a lot of conflicts. I wonder, since it's a merge of master, why do I need to revert it since the PR targets master anyway which should have those already. (scratching head)